.prpr-example.yaml

```diff
@@ -0,0 +1,47 @@
+# Put this file in your home directory
+startrek_token: your_token_here
+
```

Comment:
What's the matter of renaming this config? should we change something about setup in readme then?